src/config/configuration.ts

```diff
@@ -16,6 +19,8 @@ export interface IDatabaseConfig {
 
 export const databaseConfig = registerAs(EnumConfig.DATABASE, (): IDatabaseConfig => ({
   pg: {
+    dialect: <Dialect>process.env.SQL_DIALECT || 'postgres',
+    logging: process.env.SQL_LOGGING === 'true' || true,
```

Comment:
`process.env.SQL_LOGGING === 'true' || true`
this line of code makes no sense, try to think why.
Also, why didn't you put dialect and logging into pfConfig function?